canonical/ubuntu-pro-client
# update connectivity error message

## Why is this needed?
We want to make connectivity errors clearer and easier to debug for the user. To achieve that, we are now displaying the url that cause the error together with the cause behind the error.

Fixes: #2647

## Test Steps
1. Launch a lxd container
2. Disable network on container with: `lxc config device add NAME eth0 none`
3. Run `pro fix USN-4539-1`

Expect the following message:
```
Failed to connect to https://ubuntu.com/security/notices/USN-4539-1.json
[Errno -3] Temporary failure in name resolution 
```

## Checklist


## Does this PR require extra reviews?
